lib/ab/configuration.rb

```diff
@@ -0,0 +1,22 @@
+module Ab
+  class << self
+    attr_writer :config
+
+    def config
+      @config ||= Config.new
+    end
+
+    def configure
+      @config ||= Config.new
+      yield(@config)
+    end
+  end
+
+  class Config
+    attr_accessor :logger
+
+    def initialize
+      @logger = Logger.new(nil)
```

Comment:
@saulius oh wow, so you wrote a wrapper that does nothing to save an extra stack frame? :smile: